appium-cookies/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+build
+*.log
+.DS\_Store
```

Comment:
I think if this would be `.DS_Store` it ought to work to ignore the file. Not sure about the backslash.